src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -81,9 +83,22 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
     <Box>
       <InputLabel
         htmlFor={name}
-        required={required}
+        // To prevent asterisk from shifting far right
+        sx={{ justifyContent: showAsterisk ? 'flex-start' : undefined }}
```

Comment:
what happens if `justifyContent` is set to `undefined`?  Does it fall back to a default?

Reply:
Yep, it falls back to css styles defined higher up. So in this case it will fall back to what Odyssey has defined for this element.